staging/src/k8s.io/client-go/examples/third-party-resources/apis/tpr/v1/register.go

```diff
@@ -24,28 +24,25 @@ import (
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
+// AddToScheme applies all the functions stored in the SchemeBuilder value to a scheme.
+func AddToScheme(s *runtime.Scheme) error {
```

Comment:
It makes sense, but the caller can change SchemeBuilder anyway, so let's keep the succinct form?

We can't just make `SchemeBuilder` private because some generated functions will be moved to other packages in #44784, and they need to be able to register themselves in init.